harmony_regridding_service/regridding_service.py

```diff
@@ -787,21 +767,47 @@ def _cache_resamplers(
 
```

Comment:
Meant for L734 (`_resampled_dimensions_pairs`):

Couldn't that just be:

```
def _resampled_dimension_pairs(var_info: VarInfoFromNetCDF4) -> list[tuple[str, str]]:
    """..."""
    return [
        dims
        for dims in var_info.group_variables_by_horizontal_dimensions()
        if len(dims) == 2
    ]
```

I even bet that `pre-commit` will shift it all on to a single line.

(Sorry, I know it's out of scope, I just spotted it when checking the tests you've converted over)

Reply:
Yup, and the one below.  eb56a02

Reply:
Thanks!